### PR TITLE
Update the license of jqPlot

### PR DESCRIPTION
**Pull request by gavin0430**
ref: https://creativecommons.org/licenses/by/3.0/

### CODE REVIEW

File: ajax/libs/jqPlot/package.json [hunk 1]
```diff
@@ -12,10 +12,7 @@
   ],
   "licenses": [
     "MIT",
-    {
-      "type": "GPL",
-      "url": "http://www.opensource.org/licenses/gpl-license.php"
-    }
+    "CC-BY-3.0"
```
Comment (sufuf3) [inline ajax/libs/jqPlot/package.json]: Let's use `GPL-2.0`
The reference is https://github.com/jqPlot/jqPlot/#legal-notices.
And the commit message need to be modified (cc issue number).
Thank you.

Comment (extend1994) [reply ajax/libs/jqPlot/package.json]: There is another license: MIT
You can use (don't forget the comma)
```
"licenses": [
  "GPL-2.0"
  "MIT"
],
```

> jqPlot is currently available for use in all personal or commercial projects under both the **MIT and GPL version 2.0** licenses.

Comment (sufuf3) [reply ajax/libs/jqPlot/package.json]: @gavin0430 So please help modify this PR, thank you.

**Comment by extend1994** [comment]
Close this PR because the licenses had been updated in #11992.

**Pull request closed without merging**